storage/migrations/22_gasoutputs_height_code.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+import "github.com/go-pg/migrations/v8"
+
+// Schema version 22 adds Height and ActorName to gas outputs table
+
+func init() {
+	up := batch(`
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN height bigint;
+	ALTER TABLE public.derived_gas_outputs ADD COLUMN actor_name text;
```

Comment:
I believe constraints need to be added for these new columns since neither are null-able, and `height` is a primary key.

Reply:
Do you mean `height` should be made primary key along with `cid`?

Reply:
Please have a look to the latest commits because I am not sure I know what I'm doing with the migrations..